File: aula_0526/lista03ex01.cs
using System;

class Program {
  public static void Main() {
		Retangulo x = new Retangulo();
		
		x.SetBase(2);
		x.SetAltura(3);

		Console.WriteLine(x.GetBase());
    Console.WriteLine(x.GetAltura());
		Console.WriteLine(x.CalcArea());
		Console.WriteLine(x);



	}
}

class Retangulo {
	private double b;
	private double h;

	public void SetBase(double b){
		this.b = b;	
	}
	
	public void SetAltura(double h){
		this.h = h;
	}
	
	public double GetBase(){
			return this.b;
	}
		
	public double GetAltura(){
		return this.h;
	}
		
	public double CalcArea(){
		return b*h;
	}
		
	// public double CalcDiagonal(){
	// 	// raiz de b^2 + h^2
	// 	return	Math.Sqrt((b*b) + (h*h));
		
	// }

	public override string ToString(){
		return $"base: {b}, altura: {h}";
	}
	
	}

File: aula_0526/lista03ex01.py
import math

class Retangulo:
	def __init__(self, b, h):
		self.__b = b
		self.__h = h
	def set_base(self, b):
		self.__b = b
	def set_altura(self, h):
		self.__h = h
	def get_base(self):
		return self.__b
	def get_altura(self):
		return self.__h
	def calc_area(self):
		return (self.__b * self.__h)
	def diagonal(self):
		return (math.sqrt((self.__b**2) + (self.__h**2)))
	def __str__(self):
		return f"Base = {self.__b}, Altura = {self.__h}"


ret = Retangulo(2, 4)
print(ret.calc_area())
print(ret.diagonal())
print(ret)

ret.set_base(3)
print(ret.calc_area())
print(ret.diagonal())
print(ret)